### DISCUSSION TITLE
[Question][Hw0] Cannot Access SecuwaySSL

### DISCUSSION THREAD

**Discussion by gunsodo** [prosyslab-classroom/cs348-information-security#481] [Q&A]
I am having issues logging in to the VPN using SecuwaySSL using Apple M1. I have followed the guide both from [here](https://kcloud.kaist.ac.kr/index.php/board/?mod=document&uid=132) and [here](https://apple.secuwiz.co.kr/manual/SecuwaySSL_U_MacOS_v1.0ClientManual(20170720).pdf), but both seem to be not working. Currently, I am stuck at the *connecting* interface of the SecuwaySSL v2.0 client where I have to put in the URL, which I obtained from the [Google Sheets](https://docs.google.com/spreadsheets/d/1fxySR-T_ui_GvrmioP6hLnXXcYTZoOx2JGzyOZaUGS8/edit#gid=0) (from the `NAT IP` column). Is there any step I have missed or done incorrectly?

**Comment by rabberdabber** [comment]
Me too. It is a bit weird!

**Comment by sumango3** [comment]
Would you check if you put URL in SecuwaySSL correctly? It should be https://kcloudvpn.kaist.ac.kr/

**Comment by gunsodo** [comment]
> Would you check if you put URL in SecuwaySSL correctly? It should be https://kcloudvpn.kaist.ac.kr/

This saves the day! Thank you for correcting this. Closing this now.
